internal/builders/docker/pkg/builder.go

```diff
@@ -36,8 +36,24 @@ import (
 	"strings"
 
 	intoto "github.com/in-toto/in-toto-golang/in_toto"
+	"github.com/slsa-framework/slsa-github-generator/internal/errors"
```

Comment:
Add a space so project imports aren't lumped in with 3rd party ones.
```suggestion
	intoto "github.com/in-toto/in-toto-golang/in_toto"

	"github.com/slsa-framework/slsa-github-generator/internal/errors"
```

Reply:
Thanks. Done.